.github/workflows/tests.yml

```diff
@@ -24,7 +24,10 @@ jobs:
         # This should be the same hash as in core/tools/mavlink_router/bootstrap.py#L10
         MAVLINK_ROUTER_COMMIT="94c4e3c6a9ff7c517b20b417e232caf52f12a6c6"
         sudo apt install shellcheck
-        npm install eslint eslint-plugin-vue eslint-plugin-html --save-dev
+        npm install \
+          eslint eslint-plugin-vue eslint-plugin-html \
+          @vue/eslint-config-typescript @typescript-eslint/parser @typescript-eslint/eslint-plugin typescript \
+          --save-dev
```

Comment:
@Williangalvani I believe that we should avoid adding frontend tests in this part of the test CI, maybe creating a second job in this file? 
We could also avoid the usage of npm in the second step and use the github docker action step with the yarn docker directly.

Reply:
I agree on that, but as its the current approach, I just followed it for this PR, and we can open a following one to attack that. What do you think?

Reply:
let's talk about it. there's a lot to consider here...

Reply:
From meeting: decision is to stay as it is and do a PR focusing on refactoring our tests, so they become more atomic.